Sources/TuistKit/Services/LintImplicitImportsService.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+import Path
+import TuistCore
+import TuistLoader
+import TuistSupport
+
+enum LintImplicitImportsServiceError: FatalError {
+    case implicitImportsFound([LintingIssue])
+
+    public var description: String {
+        switch self {
+        case let .implicitImportsFound(lintingErrors):
+            "Implicit dependencies were found." + "\n" +
+                lintingErrors.map(\.reason).joined(separator: "\n")
+        }
+    }
+
+    var type: ErrorType {
+        .abort
+    }
+}
+
+final class LintImplicitImportsService {
+    private let graphImplicitLintService: GraphImplicitImportLintService
```

Comment:
The `GraphImportLintService` should be included here as a protocol, not the implementation